src/index.ts

```
@@ -255,6 +255,13 @@ export type { BaseApiClient } from './types/api';
 export { useThemeStore } from './store/themeStore';
 export type { ThemeStore, ThemeState, ThemeActions } from './store/themeStore';
 
+// Branding Store
+export { useBrandingStore } from './store/brandingStore';
+export type {
+  BrandingData,
+  BrandingState,
+} from './store/brandingStore';
```

ESLint

[error] Replace `⏎··BrandingData,⏎··BrandingState,⏎` with `·BrandingData,·BrandingState·`

(prettier/prettier)